images/linux/scripts/installers/vercel.sh

```diff
@@ -1,31 +1,31 @@
 #!/bin/bash
 ################################################################################
-##  File:  zeit-now.sh
-##  Desc:  Installs the Zeit Now CLI
+##  File:  vercel.sh
+##  Desc:  Installs the Vercel CLI
 ################################################################################
 
 # Source the helpers for use with the script
 source $HELPER_SCRIPTS/document.sh
 
-# Install the Zeit Vercel CLI
+# Install the Vercel CLI
 npm i -g vercel
 
 # Validate the installation
 echo "Validate the installation"
 if ! command -v vercel; then
-    echo "Zeit Vercel CLI was not installed"
+    echo "Vercel CLI was not installed"
     exit 1
 fi
 
-echo "Creating the symlink now to vercel"
+echo "Creating the symlink for [now] command to vercel CLI"
 ln -s /usr/local/bin/vercel /usr/local/bin/now
 
 echo "Validate the link"
 if ! command -v now; then
-    echo "Now link to Zeit Vercel CLI was not created"
+    echo "[Now] symlink to Vercel CLI was not created"
     exit 1
 fi
 
 # Document the installed version
 echo "Document the installed version"
-DocumentInstalledItem "Zeit Vercel CLI ($(vercel --version))"
+DocumentInstalledItem "Vercel CLI ($(vercel --version))"
```

Comment:
I believe tool should be documented according to this guide: https://github.com/actions/virtual-environments/blob/master/CONTRIBUTING.md#submitting-a-pull-request

> add changes to document tool name and version to the software report generator: images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1 that is used to generate software README file, e.g. Windows2019-Readme.md

Reply:
It's for windows only at the moment.

Reply:
There is no separate software report generator on linux images, and it is a common practice that we install, validate, and create documentation in the same sh script.

Reply:
oh, that's linux 🤦‍♀️. sorry for the confusion

Reply:
images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1 using in Windows deployments only